src/common/Parameter.h

```diff
@@ -466,6 +466,12 @@ class Parameter
     void create_fullname(const char *dn, char *fn, ControlGroup ctrlgroup, int ctrlgroup_entry,
                          const char *lfoPrefixOverride = nullptr) const;
 
+    // Optional OSC name
+    std::string get_OSC_name();
+    void setOSCName(const std::string &s);
+    bool hasOSCName{false};
+    std::string oscName{};
```

Comment:
No need for default construction, std::string defaults to empty. So std::string oscName is fine.